File: pset6/mario/less/mario.py
# Library
from cs50 import get_int

# Check 
while True:
    size = get_int("Height: ")
    if size > 0 and size <= 8:
        break
    
# Variables 

j = 1
z = size

# Draw
while j <= size:
    # Space
    print(" " * (z-1), end="")
    # Col
    print("#" * j)
    j += 1
    z -= 1

File: pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int size;
    do
    {
        size = get_int("Height: \n"); //Prompt
    }
    while (size < 1 || size > 8); //Repeat Prompt if size its wrong
    
    for (int j = 0; j < size; j++) 
    {
        for (int c = j + 1; c < size; c++) //Condition for space 
        {
            printf(" ");
        }
        for (int i = size - j; i <= size; i++) //Condition for #
        {
            printf("#");
        }
        printf("\n"); // Jump
    }
}
